test/e2e/test_tensorflow_inference_service.py

```diff
@@ -27,7 +27,7 @@
 from utils import wait_for_kfservice_ready
 
 api_version = constants.KFSERVING_GROUP + '/' + constants.KFSERVING_VERSION
-KFServing = KFServingClient(load_kube_config=True)
+KFServing = KFServingClient(config_file="~/.kube/config")
```

Comment:
Can we do the following logic? so that you do not need to pass in the `config_file` if loads from default location. 
```
if utils.is_running_in_k8s():
   if config_file:  load_kibe_config
   else: config.load_incluster_config()
else:
   load_kube_config
```

Reply:
@yuzisun from you suggested logic, seems still need to config default location for running in cluster case. :-)

Reply:
you are right! we only need to set this in case of using it from a remote k8s cluster, I misunderstood
 that we always need this :)